docs/topics/mta-web-downloading-an-analysis-report.adoc

```diff
@@ -17,7 +17,7 @@ For your convenience, you can download analysis reports. By default, this option
 . Click *Reports*.
 . Click the *Download report* link.
 +
-The report is downloaded and saved in the *Downloads* directory as a `.tar.gz` file.
+The report is downloaded and saved in the *Downloads* directory as a `.tar.gz` file. The name of the compressed file is the same as the application name. After you extract the file, it creates a folder with the same name as the application.
```

Comment:
```suggestion
The report is downloaded and saved in the *Downloads* directory as a `.tar.gz` file. The name of the compressed file is the same as the application name. After you extract the file, it creates a folder with the same name as the application.
```

I think using "it creates..." in a sentence without using an antecedent is awkward. Because antecedents are used to clarify what a pronoun is referring to in a sentence, and without one, a sentence may be incomplete or meaningless.